drivers/sensor/ism330dhcx/ism330dhcx.h

```diff
@@ -45,7 +45,7 @@ struct ism330dhcx_config {
 	uint8_t int_pin;
 #endif /* CONFIG_ISM330DHCX_TRIGGER */
 #if DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c)
-	uint16_t i2c_slv_addr;
+	struct i2c_dt_spec i2c;
 #elif DT_ANY_INST_ON_BUS_STATUS_OKAY(spi)
 	struct spi_dt_spec spi;
 #endif /* DT_ANY_INST_ON_BUS_STATUS_OKAY(i2c) */
```

Comment:
observation here: all instances need to be either on i2c or spi, but not both with if/elif

Reply:
Yeah, I saw that but I thought we better postpone that change to when we move all drivers to multi-instance.